GNUmakefile

```
@@ -80,9 +81,11 @@ accframework: vendor
 .PHONY: accframework
 
 acceptance: accframework
-	export TF_ACC_TEST_PATH=$(shell pwd)/internal/acceptance/vmaas/acc-prod_testcases ; \
+	export TF_ACC_TEST_PATH=$(shell pwd)/internal/acceptance/vmaas/acc-testcases ; \
 	for f in $(ACC_TEST_SERVICES); do \
 		TF_ACC=true go test -v -timeout=9000s -cover ./internal/acceptance/$$f ; \
+		# remove service tests ; \
+		rm -rf ./internal/acceptance/$$f ; \
```

I think that we want this rm to work even if there is a failure before hand.  So I'd suggest doing something similar to what we're doing in docs-generate - put a "-" in front of the TF_ACC line abovei, i.e. -TF_ACC=true ....  Test to make sure that the TF_ACC command will still run with the "-".  And maybe add a comment above the TF_ACC line to the effect that we're ignoring errors here so that follow actions will run.

We'll have to see how this works out in practice.